scripts/ci

```diff
@@ -21,6 +21,14 @@ function format_check() {
     scripts/check_copyright
 }
 
+function contracts_check() {
+    pushd zeth-contracts
+    npm config set python python2.7
```

Comment:
On my machine, the default python command points to `python3` but some of the install scripts in our package dependencies failed because they assumed python2.  This command is just to avoid that case.

Reply:
Ah, you mean why `python2.7` instead of `python2`?  I gues that would also work ... I'll check.

Reply:
Unfortunately, it seems to require the `.7` as well.

Reply:
I see, thanks